### DISCUSSION TITLE
Extracting data from different PDF versions

### DISCUSSION THREAD

**Discussion by ihsaan** [jsvine/pdfplumber#1088] [Q&A] [answered]
I created a pdf extractor which has been working fine with pdf 1.3 I recently tried the exact same document saved in pdf 1.7 and I'm picking up error. Is this expected?

**Comment by jsvine** [comment]
In theory, this shouldn't be a problem. In practice, the software that generated the 1.7 version might have introduced problems. Hard to say without seeing the PDF or the error. Does using `pdf = pdfplumber.open(..., repair=True)` fix things?

**Comment by ihsaan** [comment]
So the issue wasn't with the pdf versions but rather with my code. I used regex to find the required patterns and it's working fine now

**Comment by ihsaan** [comment, marked as answer]
So the issue wasn't with the pdf versions but rather with my code. I used regex to find the required patterns and it's working fine now
